csv-data-management/playbooks/playbooks.json

```diff
@@ -8,7 +8,7 @@
NAV1f8kZwsSgq38IrZs9G4fv1c3485MqkGxVJlCxIobn+XYcPEsigQUJCghhgXH4+SKlkMU7UANzet2zH5r7/E0rnladEMchsiI7UCz5c3CICsYwx2d8fDR48EMG5jFqE1dXERMIqgK7ZuxZVr17KUGJJdYyMjUWCo1KgOWvVoHx8pNCmqC7c4LXeYoyO6ffyx8EV1gzSIACoLzO9Z8/XXOpVMDQIgdTsCSImKEdLW2hob583LSuVUJ8XPqbRQG7QwNxcRVRn0iZ+2b6+xmP/DoUPwmj07Gy2itdOyypcti6XBwaivoY5MS4+/eFG+j1bOwJNToFUHvkEAvPT773B0ds6iEmzAZEuIqsanvBxf/ovx4yUq5uy5owrj7uSEUQMHqrUkfg+FVqWbVLmI25LRdnlICJppaMGjMsTUkN9JPxjp54ePPvhAmws0zBZmJW3n3r2oYmMjeauNlZVGnkUAP3dxwW9Xr6JcmTLZSDC36aBeveDj4qJ2YrTasX5+YkWqdIcWSOtnI1SThg3V3vtPT08h4AqAUdOn61Q/MYgFal1GlQtoLXHnz2NXdDSYhTBjUQIIfRolfHZBcRFyDs+QEOw5fDhrC0s9JT0dLRo1wj86dpSWXHVZDyP2kAkTJLvhrqAbobXqIiwUOgBVQSG9ICj0Ywo5ZmTlxEiDFAGCOfWKLVtEolLd9rQ8BpBJo0fnuoYPHj6ULOnW3bsSpMgU2CikS9teoQaQs2ZkZW1YleMRxHKWlmhsby+05VJiovhMXqMATaui5W6ZP186F3IbtDwGEUpjLzIyYFOpEjZFRqJsLs3jyvMMCiCd/N0HD9CwTh2dXk6p67I4lDOg0CoZGDhoMTlJL+9lNrIqLAwNtHRwbd21CzOioqReQkrTqkkTLJ41SyfPY1AA2ZjN/JZM36F2bSnmUExlqVLdkQge3HGbMUNtqqbL7JiW9e/eXbZ7bhofv4OiAv0jSwtjhw3DWA1N4zm/12AAMhshWWWxmzSDVIFWQhmqsrU1alWrhgE9emRTltft2IFZCxZIEZwBJCUtTd6f2zbnsStaG9UXUWJMTcViSXtaNmok/E/JpakTqp4yYp5NMYHWTJB5P5XpphqidYEBSFoyIjNBV7UGOnpGvr5du4qwWlWlGkfQWI5ctnmzOHS2ZdC3/XTsmCjISj8yF4ITb920qVTX2B7CBvCG9vZStlSVptZs345zFy/K0QQ+c96qVViyaZNI+1xkNkgyyqvjqAVGpPnFPJTDLgGleM6/cdVJbKeMHv1SZ6fqy/509KhEXnYYcFCUmBgUJFyPi8GtygL8FGdn+ZxgHjl1CvVr136JbzLVI8+kjEZF6GBMjLTJMeDQYj2dnTG8Xz9dPIRcY7AtrBRxqHiQKnDiXHF2FSwJCtL5hZULqe7sPXxYLMWybFnsXLxYLTfM+WDyQpYDGHlpuUrzEok2LZXWx1KArsMgAHJluepM1nmWjpFOyRTI4VyGDtV40kfTRGiF2378EaZGRtIT6Dx4sK5zRtiSJVi9fXtWWZM30iX0+uQTzNKz59sgALLXjxYzoHt3OU8XsWIFDhw/LtuZPo2UZLanJzq3baszCPm5UNH/ckZ+WuR79vb4sl8/ERI0nX9T/W6DABh98qRsNaU3j6R3xOTJQhn499RMMXSOj49eFbG8gki5jILFnfv3JRBxEbm1yQy4mDyyNWHkSFF+tA2DAKjuJXZHR2edr6ADpz9kQSfA3d0glugRGIhv9+8XSkSaVMfOTigPLc++dm2R2nSpDxcYgASVKvL81atlK5PXUTigb6RcNXLAAJ0kdW0Wounz7w8cAA/JdGrTRgg9yXxeRoECyBdmQWf51q3SVUBLJPejQ6eCwmwgp/Ks7yQpku4+eFCyC6UXR99n5HZ9gQPIl1u8YQMWrl8vXQZKxkCCzYyibYsWQrJJd1Q5ZG6Tothw5sIF/Hv/fhw4cQK3793DsD594O/u/iqxMywP1PbmPxw8iLClS6XCxvRO6Y8m5SGw1W1t0bRBA2lFq1m9OiqUK5eVP9N/MjAwTbxw6RLO/vqrqDNKpU4h22zIZLaji2/T9r7K54XCApWXuZacLKkbrYY/WKHIU8xzSXQVH0krZb1E6SNkRkOglfyan9N6ZRGY8aSkyL2d27VDqJeXTvTkjQRQeWkqI+x/YU5L4MjHTAhI5gUiVWX+xz/JuZHM/5RnMJ2jZdJ6WYOhsNrlo49e+VmRQmWBqqvO0iJrHD9ERyM2Ph7s0KKFGZuYCChKOsh7CCavF1KekSH6IPPm5g0bonuHDsI/1XU26GplhT6IaJsI+5svXL6MCwkJUidmTx8LTCTgBI/blbVmnh+uW6OGWByPMijig7bn5+fzQmuB2iZFBYZBQgA0M9OruVLbs/X5PF8AFtSBa30m+LqvzdeB64I68v+6QdHn+Sy/eoeG6nfkf1xAAChyMktgDwmrWcV1LNu0CXMyf3SCDUlsHFAdan90Imr1arDfj7UF1k9JcovrYB8Pj5I1dnDAuvDwl/J3tQAyTeLPPrHIM+qLL+Dm5FQs8WPGM8rHRxjB6IED4T5y5Es4qAWQfIvN2SdPnxY5nZ1MLE8Wp8FcndZHY2IBfl1EhFQWcw6NPz52+sIF6RpgilTV2hpf+/igYb16xQJDliF4PJe9OyTumn72SbKk3H7FlyVCCgAMIeUtLeE0YAD6dOki/18UB9PKE2fOYOG6dYi/dEnAY5dspL9/ts5YrUFE9QIe3OOvV0qib2QkpyybODjArlo1OYrwpg/OiUAxlTyfkICEq1dlrnRjn7RujZkeHrkajNZfsCRALNKwQM1SIZN41UT/TQdQ9f2Vxk2mjIN79xbVPOePjensA3NeSBGTlbbomBgkJiWJvMRVKgqDVmhuaopKVlbSeMRiU06+p2meOllgzpvpKyjLFxkAAdEe8/KbsHkCsChY3auaw1sA84nkWwDfAphPBPJ5+/8AHgoAtkOHtRkAAAAASUVORK5CYII=",
```

Comment:
we can replace this with the name of the images under 'images'. The framework takes care of encoding and adding the image

Reply:
hi @cs-pooja-singh  have made changes , will commit with other feedback , when  review is done